docs/source/en/_toctree.yml

```diff
@@ -1,144 +1,158 @@
-    - local: tasks/monocular_depth_estimation
-      title: Depth estimation
-    - local: tasks/image_to_image
-      title: Image-to-Image
-    - local: tasks/image_feature_extraction
-      title: Image Feature Extraction
-    - local: tasks/mask_generation
-      title: Mask Generation
-    - local: tasks/knowledge_distillation_for_image_classification
-      title: Knowledge Distillation for Computer Vision
-    title: Computer Vision
-  - isExpanded: false
+    - local: serialization
+      title: Export to ONNX
+    - local: tflite
+      title: Export to TFLite
+    - local: torchscript
+      title: Export to TorchScript
+- title: Training
+  sections:
+  - title: Trainer API
+    isExpanded: false
     sections:
-    - local: tasks/image_captioning
-      title: Image captioning
-    - local: tasks/document_question_answering
-      title: Document Question Answering
-    - local: tasks/visual_question_answering
-      title: Visual Question Answering
-    - local: tasks/text-to-speech
-      title: Text to speech
-    title: Multimodal
-  - isExpanded: false
+    - local: training
+      title: Fine-tune a pretrained model
+    - local: trainer
+      title: Trainer
+    - local: hpo_train
+      title: Hyperparameter Search using Trainer API
+    - local: run_scripts
+      title: Train with a script
+  - title: Distributed training
+    isExpanded: false
     sections:
-    - local: generation_strategies
-      title: Customize the generation strategy
-    title: Generation
-  - isExpanded: false
+    - local: accelerate
+      title: Set up distributed training with 🤗 Accelerate
+    - local: perf_train_gpu_many
+      title: Multiple GPUs and parallelism
+    - local: fsdp
+      title: Fully Sharded Data Parallel
+    - local: deepspeed
+      title: DeepSpeed
+    - local: perf_train_cpu_many
+      title: Distributed CPU training
+  - title: Hardware-specific training
+    isExpanded: false
+    sections:
+    - local: perf_train_gpu_one
+      title: Methods and tools for efficient training on a single GPU
+    - local: perf_train_cpu
+      title: Efficient training on CPU
+    - local: perf_train_tpu_tf
+      title: Training on TPU with TensorFlow
+    - local: perf_train_special
+      title: PyTorch training on Apple silicon
+    - local: perf_hardware
+      title: Custom hardware for training
+  - local: peft
+    title: Load and train adapters with 🤗 PEFT
+  - local: sagemaker
+    title: Run training on Amazon SageMaker
+  - local: debugging
+    title: Debugging
+  - local: model_memory_anatomy
+    title: Model training anatomy
+- title: Inference
+  sections:
+  - title: Pipeline API
+    isExpanded: false
+    sections:
+    - local: pipeline_tutorial
+      title: Run inference with pipelines
+    - local: pipeline_webserver
+      title: Pipelines for webserver inference
+    - local: add_new_pipeline
+      title: How to add a pipeline to 🤗 Transformers?
+  - title: LLMs
```

Comment:
A type of model that's becoming increasingly common are VLMs: they are the same as LLMs, but also accept image inputs.

Would it make sense to call this section "LLMs and VLMs"?

Reply:
For sure! Let's rename the section when we have some VLM-specific docs?